### ISSUE TITLE
Not compatible with recent julia

### ISSUE THREAD

**Issue by mloubout** [SeismicJulia/Seismic.jl#59]
Hello

`Seismic.jl` is not compatible with the fairly recent julia version `6.2.0` or any late ones due to the use of the package `Grid.jl` that is outdated and unmaintained (last update was in 2016).

**Comment by msacchi** [comment]
I noticed it I m trying to revamping all to stable 1.0 but other things got
on my way

Sent from my iPhone

On Aug 24, 2018, at 8:42 AM, Mathias Louboutin <notifications@github.com>
wrote:

Hello

Seismic.jl is not compatible with the fairly recent julia version 6.2.0 or
any late ones due to the use of the package Grid.jl that is outdated and
unmaintained (last update was in 2016).

—
You are receiving this because you are subscribed to this thread.
Reply to this email directly, view it on GitHub
<https://github.com/SeismicJulia/Seismic.jl/issues/59>, or mute the thread

**Comment by juanisabbione** [comment]
Hello Mathias,

This might have happened because you are not using the last version of
Seismic.jl. Please run *Pkg.checkout("Seismic")* to stay updated to the
last version of the repository, as stated in
https://github.com/SeismicJulia/Seismic.jl under the* Installation *
instructions.

Regards,
Juan

**Comment by juanisabbione** [comment]
Yes Mathias, you are right, the last registered version is outdated, sorry
about that.

Regards,
Juan
